src/blocks/google-ad-manager/view.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Server-side rendering of the `newspack-blocks/author-bio` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `newspack-blocks/author-bio` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the post content with latest posts added.
+ */
+function newspack_blocks_render_block_google_ad_manager( $attributes ) {
+
+	$active_ad = isset( $attributes['activeAd'] ) ? (int) $attributes['activeAd'] : 0;
+	if ( 1 > $active_ad ) {
+		return '';
+	}
+
+	$ad_code = get_post_meta( $active_ad, 'newspack_ad_code', true );
+
+	Newspack_Blocks::enqueue_view_assets( 'google-ad-manager' );
+	return $ad_code;
```

Comment:
This needs to be wrapped in a div with classes identifying the block and alignment. This is probably why @sonjaleix  is only seeing one ad per page, and why block alignment isn't working. Here's a suggested approach:

```suggestion
	return sprintf(
		'<div class="%s">%s</div>',
		esc_attr( Newspack_Blocks::block_classes( 'google-ad-manager', $attributes )),
		$ad_code
	);
```

Reply:
So, I'd already done this and hadn't committed and pushed :facepalm: